QuizApp/Views/Leaderboard/LeaderboardViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  LeaderboardViewController.swift
+//  QuizApp
+//
+//  Created by Luka Namačinski on 10.05.2021..
+//
+
+import UIKit
+
+class LeaderboardViewController: UIViewController, LeaderboardDelegate {
+    var presenter: LeaderboardPresenterProtocol!
+    
+    private var titleLabel: Label!
+    private var exitButton: UIImageView!
+    
+    private var errorMessage: Label!
+    
+    private var leaderboardView: UITableView!
+    private let cellIdentifier = "cellIdentifier"
```

Comment:
CellIdentifier bi trebao biti dio TableViewCella/CollectionViewCella jer je to **id** po kojem ti razlikuješ dva CellViewa.